gradle/deploy.gradle

```diff
@@ -9,7 +9,7 @@ publishing {
         mavenJava(MavenPublication) {
             from components.java
             groupId project.groupId
-            artifactId module_artifact_id
+            artifactId moduleArtifactId
```

Comment:
Rebase interactive your changes into the original commits - don't put the changes on top of the branch.

Reply:
Fixed in #2.